sql/core/src/main/scala/org/apache/spark/sql/execution/CacheManager.scala

```diff
@@ -108,7 +108,8 @@ class CacheManager extends Logging with AdaptiveSparkPlanHelper {
     } else {
       val sessionWithConfigsOff = getOrCloneSessionWithConfigsOff(spark)
       val inMemoryRelation = sessionWithConfigsOff.withActive {
-        val qe = sessionWithConfigsOff.sessionState.executePlan(planToCache)
+        val qe = sessionWithConfigsOff.sessionState.executePlan(
+          planToCache, shuffleCleanupMode = DoNotCleanup)
```

Comment:
isn't this the default?

Reply:
Tried to be explicit here. Removed the unnecessary argument.